<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(1, 6, 100);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>650</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="settinglisthead" native="true">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(197, 228, 243);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;</string>
           </property>
           <property name="text">
            <string>Device status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;Arial Narrow&quot;;</string>
           </property>
           <property name="text">
            <string>MONITOR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Narrow&quot;;</string>
           </property>
           <property name="text">
            <string>Become Master</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="settingVolume" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(197, 228, 243);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Narrow&quot;;</string>
           </property>
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>279</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>---------------------------------------</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btnVolumeUp">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 20.png</normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 20.png</iconset>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="settingVolumeSlider">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btnVolumeDown">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 21.png</normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 21.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="settingBrightness" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(197, 228, 243);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial Narrow&quot;;</string>
           </property>
           <property name="text">
            <string>Brightness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>279</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-----------------------------------------</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButton_automatic">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Arial Narrow&quot;;</string>
             </property>
             <property name="text">
              <string>Automatic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_manual">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Arial Narrow&quot;;</string>
             </property>
             <property name="text">
              <string>Manual</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btnBrightnessUp">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QSlider" name="settingBrightnessSlider">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btnBrightnessDown">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSettingQuit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 10.png</normaloff>../../../RPFASystem/src/viewer/res/images/PNG/资源 10.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSettingQuit</sender>
   <signal>clicked()</signal>
   <receiver>setting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
